registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/validator/impl/PacketValidatorImpl.java

```diff
@@ -177,6 +224,31 @@ private boolean biometricsXSDValidation(String id, String process, PacketValidat
 		}
 		return true;
 	}
+	
+	private String getOperationsDataFromMetaInfo(String id, String process, String fileName)
+			throws ApisResourceAccessException, PacketManagerException, IOException, JSONException, JsonParseException,
+			JsonMappingException, JsonProcessingException, io.mosip.kernel.core.util.exception.JsonProcessingException {
+		Map<String, String> metaInfoMap = packetManagerService.getMetaInfo(id, process,
```

Comment:
This method is actually performing a REST call to the packet manager, so we cannot call this multiple time.

Call this once at the beginning of biometricsXSDValidation method and pass the map to this method and biometricsSignatureValidator.validateSignature method

Reply:
called this rest service at beginning of biometricsXSDValidation and passing further...